### PR TITLE
Dt/strech feature/recommendation engine

### PR DESCRIPTION
**Pull request by DaTanUmanitoba**
Three options to choose the keywords-extraction engine:
1. use moreLikeThis only
2. use monkeyLearn API: request concurrency limited
3. keybert: slow for a single extraction, issue for load test maybe

The pull request has the option 3 and both post content and keywords are used for recommendation.

### CODE REVIEW

File: backend/util/extract-keywords.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+//another strategy: using keybert
+const { PythonShell } = require('python-shell');
+const path = require('path');
+
+async function extractKeywords(content) {
+  try{
+      const options = {
+      scriptPath: __dirname,
+      args: [content],
+      pythonPath: path.join('C://Users//52252//anaconda3//python'),
```
Comment (orangeboy55555) [inline backend/util/extract-keywords.js:10]: Will this work on a different device? What if that device does not have python installed? Do we need to add to package json?

**Review by orangeboy55555 — APPROVED** [review]
Looks good. I have one comment though.

File: backend/util/extract-keywords.js (hunk 1 above)
Comment (DaTanUmanitoba) [reply backend/util/extract-keywords.js:10]: https://stackoverflow.com/questions/38880377/npm-package-jsonengines-how-to-specify-python
we can do it like this way

**Comment by DaTanUmanitoba** [comment]
sorry, there is an updated branch for merge request based on this, I need to close it.

**Pull request closed without merging**